src/particles/diagnostics/DiagnosticOutput.cpp

```diff
@@ -141,6 +141,7 @@ namespace impactx::diagnostics
                     // preparing to access reference particle data: RefPart
                     RefPart const ref_part = pc.GetRefParticle();
 
+                    amrex::ParticleReal const s = ref_part.s;
```

Comment:
this needs to be added in two locations now:
- printed in the header (line 45)
- printed in the data table (line 156+)